src/SFA.DAS.EAS.Web/Orchestrators/EmployerAccountTransactionsOrchestrator.cs

```diff
@@ -314,6 +313,36 @@ public EmployerAccountTransactionsOrchestrator(IMediator mediator, ICurrentDateT
             };
         }
 
+        private static ICollection<TransactionLine> AggregateLevyTransactions(ICollection<TransactionLine> transactions)
+        {
+            if (transactions == null || !transactions.Any())
```

Comment:
Why use .Any()? transactions is an ICollection and so has a .Count property, which is much more efficient.

Reply:
Thats me defaulting to IEnumerable practice. Wasn't sure if the count was pre calculated. i'll change this.